docs/en/stack/ml/nlp/ml-nlp-deploy-models.asciidoc

```diff
@@ -115,36 +115,48 @@ eland_import_hub_model --url https://<hostname>:<port> --es-api-key <api-key> ..
 --------------------------------------------------
 --
 
-[discrete]
+
 [[ml-nlp-deploy-model]]
 == Deploy the model in your cluster
 
 After you import the model and vocabulary, you can use {kib} to view and manage
 their deployment across your cluster under **{ml-app}** > *Model Management*.
 Alternatively, you can use the
-{ref}/start-trained-model-deployment.html[start trained model deployment API] or
-specify the `--start` option when you run the `eland_import_hub_model` script.
+{ref}/start-trained-model-deployment.html[start trained model deployment API].
+
+You can deploy a model multiple times by assigning a unique deployment ID to 
+each deployment. It enables you to have dedicated deployments for different 
+purposes, such as search and ingest. By doing so, you ensure that the search 
+speed remains unaffected by ingest volume, and vice versa. Having separate 
+deployments for search and ingest mitigates performance issues resulting from 
+interactions between the two, which can be hard to diagnose.
```

Comment:
```suggestion
You can deploy a model multiple times by assigning a unique deployment ID when starting
the deployment. It enables you to have dedicated deployments for different 
purposes, such as search and ingest. By doing so, you ensure that the search 
speed remains unaffected by ingest workloads, and vice versa. Having separate 
deployments for search and ingest mitigates performance issues resulting from 
interactions between the two, which can be hard to diagnose.
```

Reply:
Plus can we add something about how the queue works

## Inference Request Queues
Every allocation of a model deployment has a dedicated queue to buffer inference requests. The size of this queue is controlled by the queue_capacity parameter to the start model deployment API. Once the queue is full any new requests will be rejected until some of the queued requests have been processed and there is capacity once again. Where multiple ingest pipelines reference the same model deployment it is possible for the queue to fill up resulting in rejected requests. Consider using dedicated deployments in this situation. 
 
Inference requests originating from search such as the text_expansion query or the query_vector_builder option in knn search are prioritised over non search requests.  The inference ingest processor creates normal priority requests, if a search query and ingest processor are using the same model deployment the higher priority search requests skip to the front of the queue to be processed before the lower priority ingest requests. This makes search responses lively while slowing down ingest where response time is not critical.